flags/mod.ts

```diff
@@ -201,6 +208,22 @@ export function parse(
     }
   }
 
+  if (negatable !== undefined) {
+    const negatableArgs = typeof negatable === "string"
```

Comment:
Does it make sense to perform some input validation here? What if `negatable` is not a string, but also not an array? Or, what if it is an array that includes things besides strings. Or, what if the provided values are strings, but nonsense values? It feels like we should either have validation, tests, or both for these edge cases.